lua/tabline/highlights.lua

```diff
@@ -41,19 +41,22 @@ M.c = {
     inactive_bg = M.get_color('TabLine', 'bg'),
     active_text = '#eeeeee',
     inactive_text = '#7f8490',
-    active_sep = '#ff6077',
+    active_sep = '#87ceeb',
+    modified_active_sep = '#ff6077'
```

Comment:
Make both of these to the color '#ff6077'

Reply:
Also add a trailing comma after the last line (or you could format with the stylua config I have 😄)

Reply:
I did not know about stylua but now I know it and also fixed this.